src/gui/MatchTemplatePanel.cpp

```diff
@@ -1143,18 +1178,20 @@ void MatchTemplatePanel::ResumeRunCheckBoxOnCheckBox(wxCommandEvent& event) {
     }
 }
 
-void MatchTemplatePanel::CheckForUnfinishedWork(bool is_checked, bool is_from_check_box) {
+wxArrayLong MatchTemplatePanel::CheckForUnfinishedWork(bool is_checked, bool is_from_check_box) {
     // This may be called when the user clicks the resume run checkbox.
     // OR
     // When the header in the results panel is changed.
-
+    wxArrayLong unfinished_match_template_ids;
```

Comment:
Would this be an okay time to not ad more wxArrays? There was a function above that returned wxArrayLong - could we just overload it to return a std::array or std::vector<long>?

Reply:
I agree, but I couldn't figure out how to do it in a few minutes...

It sounds like in newer wx version wxArrayLong is a synonym for st::array<long> anyways.

Reply:
Okay, let's leave for now and test wx 3.1 in the dev container